src/annotation.js

```diff
@@ -98,6 +98,18 @@ function resolveAnimations(chart, animOpts, mode) {
 	return new Animations(chart, animOpts);
 }
 
+function initElement(chart, element, elementType, options, animations) {
```

Comment:
Sorry, can you revert this whole thing? Lets not move the `_display` to `options.display` or name this function `initElement`, because its done multiple times to same element, so its not initializing. Also lets keep it below `updateElements`, for easier reading of the file.

Reply:
OKI! Done! I'm gonna push in next minutes

Reply:
@kurkle Now I remember why I have created another function.
ESLINT:
```javascript
101:1  error  Function 'updateElements' has a complexity of 11. Maximum allowed is 10  complexity
```

I can remove the check if `_display` is `true` but in my opinion that's not correct. 
What do you think?

Reply:
oh, we have that strict rules. I'll update the eslint config.